wikitongues/wikitongues/data_store/airtable/airtable_table_info.py

```diff
@@ -11,10 +14,15 @@ def __init__(self, name, id_column, page_size=100, max_records=100):
             name (str): Table name
             id_column (str): Name of identifier column
             page_size (int, optional): Number of records returned per request
+            offset (str): Offset value for pagination
             max_records (int, optional): Max records to retrieve
         """
 
         self.name = name
         self.id_column = id_column
         self.page_size = page_size
+        self.offset = None
+        if os.path.exists(os.path.expanduser('~/.language-indexing-offset')):
+            file = open(os.path.expanduser('~/.language-indexing-offset'), 'r')
+            self.offset = file.read()
```

Comment:
I'd like to move the file reading somewhere else, for better [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns):
* Create a new class containing a function that reads the file and returns the value
* Call this function from language_indexing.py
* Pass the returned offset value into the AirtableTableInfo constructor

Reply:
Sounds good, should the new class be in a new .py file?

Reply:
Yes please!